modules/rst.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+"""
+Add an |rst filter to the jinja templates…
+"""
+
+from functools import partial
+
+from docutils.core import publish_parts
+from jinja2 import Markup
+
+from typogrify import typogrify_filter
+
+
+def rst_filter(config, s):
+    settings = {
+        'initial_header_level': config.get('rst_header_level', 2),
+        'smart_quotes':         config.get('smart_quotes', False),
+        'syntax_highlight':     'short'
```

Comment:
Super small style gripe: I prefer one space between `:` and the value, rather than aligning values.

Reply:
(This is not even a little blocking.)

Reply:
Yes, agreed. I copy-pasted some of this from rstblog, and neglected to correct for style.